cooperator_worker/README.rst

```diff
@@ -0,0 +1,59 @@
+===============================
+Beescoop link with easy my coop
```

Comment:
@victor-champonnois Will the strings "easy my coop" in readme be changed later ? Otherwise, best to update the readme now. (see ligne 22 as well)

Reply:
merci 👍